doc/rados/operations/health-checks.rst

```diff
@@ -712,13 +724,14 @@ This warning can be disabled with:
 BLUESTORE_DISK_SIZE_MISMATCH
 ____________________________
 
-One or more OSDs using BlueStore has an internal inconsistency between the size
-of the physical device and the metadata tracking its size.  This can lead to
-the OSD crashing in the future.
+One or more BlueStore OSDs have an internal inconsistency between the size of
+the physical device and the metadata that tracks its size. This inconsistency
+can lead to the OSD(s) crashing in the future.
 
-The OSDs in question should be destroyed and reprovisioned.  Care should be
-taken to do this one OSD at a time, and in a way that doesn't put any data at
-risk.  For example, if osd ``$N`` has the error:
+The OSDs that have this inconsistency should be destroyed and reprovisioned. Be
+very careful to execute this procedure on only one OSD at a time, so as to
+minimize the risk of losing any data. To execute this procedure, where ``$N``
+is the OSD that has the inconsistency, run the following commands:
```

Comment:
Maybe add a note about waiting for recovery to complete before proceeding to the next.